.luacheckrc

```diff
@@ -48,4 +48,5 @@ exclude_files = {
   'runtime/lua/vim/_meta/api.lua',
   'runtime/lua/vim/re.lua',
   'src/nvim/eval.lua',
+  'src/nvim/options.lua',
```

Comment:
Do we definitely need to ignore all these files? Generated code can have errors.

Reply:
Yep, luacheck provides way too many false-negatives in `[[...]]` strings.

Reply:
Also this file isn't generated.

Reply:
Maybe, but honestly I've never found luacheck that valuable.

I'll have a look.

Reply:
hmm. created https://github.com/neovim/neovim/issues/24563

Reply:
done